spec.html

```diff
@@ -29809,7 +29809,8 @@
           1. Let _n_ be ? ToInteger(_fromIndex_). (If _fromIndex_ is *undefined*, this step produces the value 0.)
           1. If _n_ &ge; _len_, return -1.
           1. If _n_ &ge; 0, then
-            1. Let _k_ be _n_.
+            1. Let _k_ be _n_ + (*+0*).
+            1. NOTE: Adding a positive zero converts *-0* to *+0*.
```

Comment:
Okay. I'll also update [20.3.1.15 TimeClip](http://tc39.github.io/ecma262/#sec-timeclip) to use an extra step.